.github/workflows/testAndPublish.yml

```diff
@@ -197,7 +197,12 @@ jobs:
       run: ci/scripts/setBuildVersionVars.ps1
     - name: Set scons args
       run: ci/scripts/setSconsArgs.ps1
-    - name: Prepare source code for launcher
+      env:
+        apiSigningToken: ${{ (github.event_name == 'push' && env.feature_signing) && secrets.API_SIGNING_TOKEN || '' }}
+    - name: Install SignPath
+      if: ${{ github.event_name == 'push' && env.feature_signing }}
+      run: Install-Module -Name SignPath -Force
```

Comment:
Could you elaborate on why Install-PSResource is not working here?
Note that Install-Module is ,old and slow, install-psresource is its successor.

Reply:
I think I resolved this in 81767ef233ad9104c675f1bb1f7b0ceb76c947c9